src/azure-cli/azure/cli/command_modules/network/custom.py

```diff
@@ -3023,14 +3023,24 @@ def check_nw_ip_flow(cmd, client, vm, watcher_rg, watcher_name, direction, proto
                      resource_group_name=None, nic=None, location=None):
     VerificationIPFlowParameters = cmd.get_models('VerificationIPFlowParameters')
 
-    local_ip_address, local_port = local.split(':')
-    remote_ip_address, remote_port = remote.split(':')
+    try:
+        local_ip_address, local_port = local.split(':')
+        remote_ip_address, remote_port = remote.split(':')
```

Comment:
If you want to judge if ```:``` is provided in local or remove variable, split(':') may not work. Because it do not cause error when there is no ```:``` in string. I think contains() or other functions can help you, or you can do a more detailed check for the format.

Reply:
I think it will cause error when there is no `:` in the string. #9844